plugin/path_checkouts.go

```diff
@@ -74,3 +408,20 @@ func (b *backend) operationReserveStatus(ctx context.Context, req *logical.Reque
 		Data: respData,
 	}, nil
 }
+
+func canCheckIn(req *logical.Request, checkOut *CheckOut, disableCheckInEnforcement bool) bool {
+	if disableCheckInEnforcement {
+		return true
+	}
+	if checkOut.BorrowerEntityID != "" && req.EntityID != "" {
+		if checkOut.BorrowerEntityID == req.EntityID {
+			return true
+		}
+	}
+	if checkOut.BorrowerClientToken != "" && req.ClientToken != "" {
+		if checkOut.BorrowerClientToken == req.ClientToken {
+			return true
+		}
```

Comment:
Can use: `return checkOut.BorrowerClientToken == req.ClientToken`

Reply:
We could at this point, but I actually think it reduces code clarity and is more of a personal stylistic preference?

Reply:
It's ok as is.